prime-dotnet-webapi/Services/Rules/AutomaticAdjudicationRules.cs

```diff
@@ -278,22 +281,32 @@ public override async Task<bool> ProcessRule(Enrollee enrollee)
                 if (paperEnrolleeMatchId == -1)
                 {
                     enrollee.AddReasonToCurrentStatus(StatusReasonType.PaperEnrolmentMismatch, $"User-Provided GPID: {potentialPaperEnrolleeGpid}");
+                    enrollee.AddReasonToCurrentStatus(StatusReasonType.PossiblePaperEnrolmentMatch, $"birthdate matches enrolment(s): {paperEnrolleeIdsAsString}");
                     return false;
                 }
-                // if match link to paper enrolment and confirm the linkage here, if failed to link we add status reason.
+                // if a match is found, link to paper enrolment and confirm the linkage here, if failed to link we add status reason.
                 if (!await _enrolleePaperSubmissionService.LinkEnrolleeToPaperEnrolmentAsync(enrolleeId: enrollee.Id, paperEnrolleeId: paperEnrolleeMatchId))
                 {
                     enrollee.AddReasonToCurrentStatus(StatusReasonType.UnableToLinkToPaperEnrolment, $"User-Provided GPID: {potentialPaperEnrolleeGpid}");
+                    enrollee.AddReasonToCurrentStatus(StatusReasonType.PossiblePaperEnrolmentMatch, $"birthdate matches enrolment(s): {paperEnrolleeIdsAsString}");
                     return false;
                 }
                 return true;
             }
             // if yes and GPID not provided - flag with "Possible match with paper enrolment"
             else
             {
-                enrollee.AddReasonToCurrentStatus(StatusReasonType.PossiblePaperEnrolmentMatch);
+                enrollee.AddReasonToCurrentStatus(StatusReasonType.PossiblePaperEnrolmentMatch, $"birthdate matches enrolment(s): {paperEnrolleeIdsAsString}");
                 return false;
             }
         }
+
+        static private string CreateIdsString(IEnumerable<Enrollee> paperEnrollees)
+        {
+            var paperEnrolleeIdsList = paperEnrollees
+                .Select(pe => pe.Id);
+
+            return string.Concat(string.Join(", ", paperEnrolleeIdsList), ".");
```

Comment:
why call `string.Concat` when you can just explicitly add a period to the status reasons?

Reply:
Fixed, completely skipped my mind lol